### PR TITLE
[suggestion] feat: Add NewManifestMode config

### PR DESCRIPTION
**Pull request by michaljurecko**
See: https://github.com/keboola/processor-split-table/pull/34

### CODE REVIEW

File: internal/pkg/processor/config/config.go [hunk 1]
```diff
@@ -91,12 +100,21 @@ func processValidateError(err validator.ValidationErrors) error {
...
-			e.Value(),
-			e.ActualTag(),
-		)
+		if e.ActualTag() == "oneof" {
+			msg += fmt.Sprintf(
+				`unexpected value "%s" for "%s", use one of "%s"`,
+				e.Value(),
+				path,
+				strings.Join(strings.Split(e.Param(), " "), `", "`),
+			)
```
Comment (michaljurecko) [inline internal/pkg/processor/config/config.go:109]: :one:

File: internal/pkg/slicer/config/config.go [hunk 2]
```diff
@@ -126,84 +122,10 @@ func (v *Config) UnmarshalJSON(data []byte) error {
...
-	switch str {
-	case "bytes":
-		*m = ModeBytes
-	case "rows":
-		*m = ModeRows
-	case "slices":
-		*m = ModeSlices
-	default:
-		return fmt.Errorf(`unexpected value "%s" for "mode", use "rows", "bytes" or "slices"`, str)
-	}
```
Comment (michaljurecko) [inline internal/pkg/slicer/config/config.go:167]: :one:

File: internal/pkg/processor/config/config_test.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ func getTestData() []testData {
 		{
 			comment:  "invalid mode",
 			input:    "{\"parameters\": {\"mode\": \"abc\"}}",
-			error:    `invalid configuration: unexpected value "abc" for "mode", use "rows", "bytes" or "slices"`,
+			error:    `invalid configuration: unexpected value "abc" for "parameters.mode", use one of "bytes", "rows", "slices"`,
 			expected: nil,
```
Comment (michaljurecko) [inline internal/pkg/processor/config/config_test.go:58]: :one:

**Review by michaljurecko — COMMENTED** [review]
This is probably the only thing that may not be clear, this belongs together:

**Review by jachym-tousek-keboola — APPROVED** [review]

**Pull request merged**